lib/package-managers/npm.js

```diff
@@ -107,25 +107,60 @@ module.exports = {
             npm.commands.listAsync(args || [], true); // silent:true
     },
 
-    latest: cint.partialAt(view, 1, 'dist-tags.latest'),
+    latest: function (packageName, currentVersion, includePrerelease) {
+        return view(packageName, 'dist-tags.latest')
+            .then(function (version) {
+                if (
+                    versionUtil.getPrecision(version) !== 'release' ||
+                    (
+                        versionUtil.getPrecision(version) === 'release' &&
```

Comment:
You can omit line 116. It is guaranteed to be true here since you've already checked if it's not "prerelease" in the other branch of the OR.